shared/panels/export-csv-sidepanel/export-csv-sidepanel.tsx

```diff
@@ -0,0 +1,87 @@
+import { Button } from "@/design-system/atoms/button/variants/button-default";
+import { DateRangePicker } from "@/design-system/atoms/date-range-picker";
+import { Paper } from "@/design-system/atoms/paper";
+import { Typo } from "@/design-system/atoms/typo";
+import { CheckboxButton } from "@/design-system/molecules/checkbox-button";
+
+import { SidePanelBody } from "@/shared/features/side-panels/side-panel-body/side-panel-body";
+import { SidePanelFooter } from "@/shared/features/side-panels/side-panel-footer/side-panel-footer";
+import { SidePanelHeader } from "@/shared/features/side-panels/side-panel-header/side-panel-header";
+import { useSidePanel } from "@/shared/features/side-panels/side-panel/side-panel";
+import { Translate } from "@/shared/translation/components/translate/translate";
+
+import { useExportCsvSidepanel } from "./export-csv-sidepanel.hooks";
+import { ExportCsvSidepanelProps } from "./export-csv-sidepanel.types";
+
+export function ExportCsvSidepanel({
+  types,
+  dateRange,
+  onDateRange,
+  exportedData,
+  onExport,
+  onClose,
+}: ExportCsvSidepanelProps) {
```

Comment:
Need `canGoBack` prop as the panel can be opened from a page, not just a sidepanel.